server/static/js/index/home_page_monitor_button.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+$(document).ready(function () {
+    /**
+     * Open device monitor panel on new tab
+     */
+    $(table.device.button.monitor).click(function () {
+        let URL = '/deviceStatus/' + $(tableRowsHomePage.active.device).children('[data-device-type-id]').text();
+
+        window.open('http://127.0.0.1:5000' + URL, '_blank')
```

Comment:
Didn't notice it the first time. Consider removing the full URL - relative path using URL variable only is plenty enough

Reply:
While we're at it, do the same in home_page_debug_button.js file too.